jsonparser.cpp

```diff
@@ -0,0 +1,62 @@
+#include "jsonparser.h"
+#include <sstream>
+
+std::map<std::string,std::string> JsonParser::ParseFile(const std::string& filename){
+	std::map<std::string, std::string> unit;
+	std::ifstream input;
+	input.open(filename);
+	if (input.fail()){
+		throw 1;
```

Comment:
Hát, ettől nem lesz sokkal okosabb, aki látja.

Reply:
Úgy látom, nem voltam elég egyértelmű. Nem a változók nevei zavartak, hanem a throw 1-ben az 1. Egy hónap múlva senki nem fog emlékezni, mi váltja ki az 1-es típusú hibát, meg mi a 3497-eset. Azon kívül, hogy a kivétel típusát is lehetne valami alkalmasabbra választani, a számokat is lehetne pl. egy beszédesebb nevű enum konstanssal helyettesíteni.